docs/redux/redux-intro.md

```diff
@@ -163,26 +242,23 @@ function counter (state = 0, action) {
 			return state;
 	}
 }
```

Comment:
Let's add one empty line after 140 line.

Reply:
Let's add `;` to https://github.com/enactjs/enact/pull/3081/files#diff-8a5a91a87d99a04436c951e212abd293a95e1164030ac87952439038334d26e5R178.